web/locales/pt-br.yml

```diff
@@ -26,18 +27,20 @@
   Delete: Apagar
   AddToQueue: Adicionar à fila
   AreYouSureDeleteJob: Deseja deletar esta tarefa?
-  AreYouSureDeleteQueue: Deseja deletar a %{queue} fila?
+  AreYouSureDeleteQueue: Deseja deletar a fila %{queue}? Isso irá deletar todos as tarefas desta fila, e irá reaparecer caso mais tarefas sejam adicionadas à mesma no futuro.
```

Comment:
I think this is too long. I'd remove the second sentence.

Reply:
Thanks for the feedback. Suggestion applied! 🙂